sherpa/astro/io/__init__.py

```diff
@@ -1206,14 +1225,14 @@ def _make_float32_array(rows, ncols):
     """
 
     nrows = len(rows)
-    out = numpy.zeros((nrows, ncols), dtype=numpy.float32)
+    out = np.zeros((nrows, ncols), dtype=np.float32)
     for idx, row in enumerate(rows):
         out[idx, 0:len(row)] = row
 
     return out
```

Comment:
This is fine, but for my own education, I looked around for other implementations of the same problem and came upon this:
```Python
import itertools
return np.array(list(itertools.zip_longest(*v, fillvalue=0))).T
```
(from https://stackoverflow.com/questions/38619143/convert-python-sequence-to-numpy-array-filling-missing-values)

I like the existing Sherpa code better (it's easier to read), but I thought I'll just comment on what I learned - I probably don't look at `itertools` often enough.